### ISSUE TITLE
Auto Import path is broken

### ISSUE THREAD

**Issue by jbeckton** [Microsoft/vscode#38453]
Just installed VSCode on a new Mac a month ago and this auto import issue has been a problem every since. I cannot find a way to fix and it's about to drive me back to WebStorm.

I am using Typescript and Angular, when I enter a component class name in code I get the req underline if that class has not been imported, if I select the text I get the bulb icon to the left and it offers to import the missing class but the path is wrong as well as it's using double quotes when my config says to use single quotes. 

the path I am seeing is "src/app/folder/subfolder/component.name"

So what do I need to do to fix this problem?

**Comment by mjbvz** [comment]
Please describe what the expected behavior is. What does your project structure look like and what is the expected import path?

**Comment by jbeckton** [comment]
@mjbvz Project structure is kind of irrelevant, I expect the path to be relative to the file having the import which is what Typescript usually expects. There are several auto import extensions for VSC that provide the correct path but the one I have just installed adds the new import to the top of the other imports which is useless to me since I then have to move it to the bottom.

The auto import stuff has worked fine up until a month ago when I installed VSC on a new Mac. I use VSC at home and on other work machines with no issues, I'm trying to figure out how to correct this on my new machine.
